packages/front-end/components/Experiment/TabbedPage/ExperimentDecisionExplanation.tsx

```diff
@@ -0,0 +1,184 @@
+import { Box, Text, Flex } from "@radix-ui/themes";
+import { DecisionFrameworkExperimentRecommendationStatus } from "back-end/types/experiment";
+import {
+  DecisionCriteriaInterface,
+  DecisionCriteriaRule,
+} from "back-end/src/enterprise/routers/decision-criteria/decision-criteria.validators";
+import Link from "@/components/Radix/Link";
+
+interface Props {
+  status: DecisionFrameworkExperimentRecommendationStatus;
+  variations: {
+    variationId: string;
+    decidingRule: DecisionCriteriaRule | null;
+  }[];
+  variationNames: Record<string, JSX.Element>;
+  showDecisionCriteria: boolean;
+  setShowDecisionCriteria: (show: boolean) => void;
+  showDecisionCriteriaLink?: boolean;
+}
+
+const getRecommendationText = (
+  status: "ship-now" | "ready-for-review" | "rollback-now",
+  multipleVariations: boolean
+) => {
+  switch (status) {
+    case "ship-now":
+      return `Review reasons to ship${
+        multipleVariations ? " and select the preferred variation" : ""
+      }.`;
+    case "ready-for-review":
+      return `Reasons to review${
+        multipleVariations ? " the eligible variations" : ""
+      }.`;
+    case "rollback-now":
+      return "Review reasons to rollback.";
+  }
+};
+
+export default function ExperimentDecisionExplanation({
+  status,
+  variations,
+  variationNames,
+  showDecisionCriteria,
+  setShowDecisionCriteria,
+  showDecisionCriteriaLink,
+}: Props) {
+  // If the experiment is not in a decision state, don't show anything
+  if (
+    status.status !== "ship-now" &&
+    status.status !== "ready-for-review" &&
+    status.status !== "rollback-now"
+  ) {
+    return null;
+  }
+
+  // fix to get deciding rule
+  const getConditionText = (
+    condition: DecisionCriteriaInterface["rules"][0]["conditions"][0]
+  ) => {
+    const metricType = (() => {
+      switch (condition.metrics) {
+        case "goals":
```

Comment:
nit: should the returned strings be capitalized?

Reply:
I don't think so? They don't start the sentence. I'm not sure we always capitalize them mid-sentence.